index.html

```diff
@@ -51,22 +56,203 @@ <h1 id="principal-title" class="color-blue fonts">I’m Luis Emilio. <br> Glad t
                     <a href="https://github.com/lerfast"><img src="./vectors/github logo.png" alt="github-icon"></a>
                 </li>
                 <li>
-                    <a href="https://www.angellist.com/"><img src="./vectors/angellist-icon.png" alt="angellist-icon"></a>
+                    <a href="https://www.angellist.com/"><img src="./vectors/angellist-icon.png"
+                            alt="angellist-icon"></a>
                 </li>
-        
+
             </ul>
         </div>
-            
-            
-
-            
-                
-                
-            
-            
-         
-           
-        
     </main>
+    <section class="works-container">
+
+        <div class="grid-container">
+        
+            <img class="card-img" src="./vectors/Snapshoot Portfolio.png" alt="image of my project" />
+            <div class="text-container">
+                <h2 class="card-title">Tonic</h3>
+                    <div class="frame-2">
+                        <p class="client">CANOPY</p>
+                        <img src="./vectors/Counter.png" alt="Counter">
+                        <p class="Role">Back End Dev</p>
+                        <img src="./vectors/Counter.png" alt="Counter">
+                        <p class="Role">2023</p>
+                    </div>
+                    <p class="card-description">
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+        
+        
+                    <ul class="tags-list-style-none">
+                        <li class="tag-description1">HTML</li>
+                        <li class="tag-description2">CSS</li>
+                        <li class="tag-description3">JAVASCRIPT</li>
+                    </ul>
+                    <button type="button" class="see-btn">See project</button>
+            </div>
+        </div>   
+
+            <div class="grid-container">
+                <div class="card-container">
+                    <img class="card-img" src="./vectors/Snapshoot Portfolio2.png" alt="image2 of my project" />
```

Comment:
- [ ] Nice work on the cards, however, they look slightly different that the Figma design because they lack padding 🔽, Please work on that.

## Your Design
![2023-03-24_230250](https://user-images.githubusercontent.com/104319462/227659996-3c46a3a7-d29b-4d79-8003-2f009b13d92f.png)

## Figma Design  
![2023-03-24_230346](https://user-images.githubusercontent.com/104319462/227659989-8bacf931-3a4c-49f1-bd16-0ff7a1b0209c.png)